### ISSUE TITLE
0 HP shown as Dead instead of lowest Stage

### ISSUE THREAD

**Issue by yjeroen** [mclemente/healthEstimate#127]
Module: Health Estimate
System: PF2e
Issue: Some characters show as "Dead" at 0 HP, while I'd expect the lowest state to be "Dying".

![image](https://user-images.githubusercontent.com/1303988/220448860-9940fb44-7931-44f2-bdc1-51f26aac0e80.png)

**Comment by jacobwojoski** [comment]
Did you look at the NPCorPLAYER->Token->Health Estimate Settings

**Comment by yjeroen** [comment]
> Did you look at the NPCorPLAYER->Token->Health Estimate Settings

That setting doesn't seem to have any effect for me.

![image](https://user-images.githubusercontent.com/1303988/220452159-5c4936d4-9937-407a-a9f0-b4766ea44683.png)

**Comment by jacobwojoski** [comment]
What are you trying to do exactly? Did you Uncheck the NPC's die immediately?

**Comment by yjeroen** [comment]
[health-estimate-issue.webm](https://user-images.githubusercontent.com/1303988/220462086-cd6326ee-5880-4dc1-afe2-ab54b773ea4b.webm)

A video shows much more than words. :)

**Comment by yjeroen** [comment]
Ah I found it:
![image](https://user-images.githubusercontent.com/1303988/220463384-61ee2413-83f3-4a3e-88a0-0c9b057849e4.png)

This setting thinks PF2e Player Characters are "NPC's".

Current workaround via:
![image](https://user-images.githubusercontent.com/1303988/220463546-b4f2faca-f9f7-4be7-9417-419463dc67db.png)

And then if during Combat, an NPC gets dropped to 0, the pf2e system adds the Dead status to the NPC, so the "Dead" state gets shown. Players don't get the Dead status automatically, so they show as dying:

![image](https://user-images.githubusercontent.com/1303988/220463842-182fdc7b-a38c-444c-8a71-5c8e9e33627e.png)

**Comment by yjeroen** [comment]
Preferred fix would be for the module to recognize PF2e player characters as non-NPC's.

**Comment by mclemente** [comment]
Each system can have its own nomenclature for character/npc, so it is safer for the module to assume a PC is an actor that's been selected by a player.
Right click a player's name on the players' list > User Configuration > assign a character to them.
![image](https://user-images.githubusercontent.com/5288872/220517460-b0a739f0-2fbb-4695-bcd5-aeb35f9f7e34.png)

**Comment by mclemente** [comment]
"NPC" in the `NPCs die immediately` setting is the shortest and simplest term for "actors who aren't selected by players", but I'd need to explain what "selected" means, which would make the hint super long and might not be clear at a glance for most users. Best I can do is point out the [Foundry article](https://foundryvtt.com/article/users/) to explain what it means.

I changed the description, but the behavior will remain the same. Closing this as wontfix.
